select.scss

```diff
@@ -0,0 +1,127 @@
+// Select
+// ===
+
+
+// Configurable Variables
+// ---
+
+// Whether to draw the default down-arrow icon.
+$select__draw-caret: true !default;
+
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'spline/dist/utilities/triangle';
+@import 'level/level';
+
+
+// Root
+// ---
+
+.c-select {
+    position: relative; // Allow positioning of select element and caret
+
+    display: block;
+    box-sizing: border-box;
+    max-width: 100%;
+    min-height: $tap-size;
+    border: 1px solid $border-color;
+
+    border-radius: $border-radius;
+
+    font-family: inherit;
+
+    user-select: none;
+
+
+    // Caret
+    // ---
+
+    @if ($select__draw-caret) {
+        &::after {
+            @include triangle(
+                $type: 'triangle',
+                $direction: 'down',
+                $size: 7px,
+                $color: $border-color
+            );
+
+            content: '';
+
+            position: absolute;
+            top: 0;
+            right: $horizontal-input-padding;
+            bottom: 0;
+
+            margin: auto; // Vertically center the caret
+        }
+    }
+
+    // Select element
+    // ---
+
+    select {
+        position: absolute;
+        top: 0;
+        left: 0;
+
+        width: 100%;
+        height: 100%;
+        padding-right: $horizontal-input-padding;
+        padding-left: $horizontal-input-padding;
+        border: 0;
+
+        background: none;
+
+        -webkit-appearance: none;
+    }
+}
+
+// Select with Inner Label
+// ---
+
+.c-select.c--inner-label {
+    display: flex;
+    align-items: center; // vertically align inner label and value
```

Comment:
Do we want to be consistent with our comment style? In which case, comments should follow the format...

```scss
// Select with Inner Label
// ---
//
// 1. vertically align inner label and value

.c-select.c--inner-label {
    align-items: center; // 1
}
```

and etc.